azure_arc_data_jumpstart/eks/terraform/artifacts/DataServicesLogonScript.ps1

```diff
@@ -1,27 +1,32 @@
 Start-Transcript -Path C:\Temp\DataServicesLogonScript.log
 
 # Deployment environment variables
-$connectedClusterName="Arc-Data-EKS-K8s"
+$Env:TempDir = "C:\Temp"
+$connectedClusterName = "Arc-Data-EKS"
 
 Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False
 
-$azurePassword = ConvertTo-SecureString $env:spnClientSecret -AsPlainText -Force
-$psCred = New-Object System.Management.Automation.PSCredential($env:spnClientId , $azurePassword)
-Connect-AzAccount -Credential $psCred -TenantId $env:spnTenantId -ServicePrincipal
-
 # Login as service principal
-az login --service-principal --username $env:spnClientId --password $env:spnClientSecret --tenant $env:spnTenantId
+az login --service-principal --username $Env:spnClientId --password $Env:spnClientSecret --tenant $Env:spnTenantId
 
-# Installing Azure CLI arcdata extension
+# Making extension install dynamic
+az config set extension.use_dynamic_install=yes_without_prompt
+# Installing Azure CLI extensions
 Write-Host "`n"
```

Comment:
Matt added a "Write-Header" method in ArcBox that we had been using to make log output look cleaner. Can we use that here as well?